text/0401-coercions.md

```diff
@@ -321,18 +321,26 @@ following holds:
 
 * `e` has type `T` and `T` coerces to `U`;
 
-* `e` has type `*T` and `U` is `*U_0` (i.e., between any raw pointers);
+* `e` has type `*T`, `U` is `*U_0`, and either `U_0: Sized` or
+    unsize_kind(`T`) = unsize_kind(`U_0`);
```

Comment:
It throws away the DST info - that's a perfectly good cast (same as `u32 -> u8`).